dds/DCPS/transport/rtps_udp/RtpsUdpSendStrategy.cpp

```diff
@@ -713,6 +713,22 @@ RtpsUdpSendStrategy::stop_i()
 {
 }
 
+size_t RtpsUdpSendStrategy::max_message_size(TransportQueueElement* element) const
+{
+  size_t size = max_message_size_;
+#ifdef OPENDDS_SECURITY
+  if (element) {
+    // Temp Dummy Fixed Value. This should be enough for the PREFIX (24) and
+    // POSTFIX (24), and the bytes being added to the payload, but it doesn't
+    // seem to be working.
+    size -= 100;
```

Comment:
I think size can be zero, meaning unlimited. Better check for that here, or you'll get negative / underflow sizes.

Reply:
I know that's valid for tcp, but is it valid for rtps?

Reply:
It defaults to a big number, but it's also configurable so it can be whatever a user wants to make it.